Token.js

```diff
@@ -268,8 +266,59 @@ class Token {
 
 	}
 
+	/**
+	 * adds a hidden dead cross to tokens
+	 * makes dead cross visible if token has 0 hp
+	 * @param token jquery selected div with the class token
+	 */
+	update_dead_cross(token){
+		if(this.options.max_hp > 0){
+			// add a cross if it doesn't exist
+			if(token.find(".dead").length === 0) 
+				token.prepend(`<div class="dead" hidden></div>`);
+			// update cross scale
+			const deadCross = token.find('.dead')
+			deadCross.attr("style", `transform:scale(${this.get_token_scale()});--size: ${parseInt(this.options.size) / 10}px;`)
+			// check token death
+			if (this.options.max_hp > 0 && parseInt(this.options.hp) === 0) {
+				deadCross.show()
+			} else {
+				deadCross.hide()
+			}
+		}
+	}
+
+	/**
+	 * updates the colour of the health aura if enabled
+	 * @param token jquery selected div with the class token
+	 */
+	update_health_aura(token){
+		console.group("update_health_aura")
+		if ((!this.options.disableaura) && this.options.max_hp > 0) {
```

Comment:
You don't need the extra parentheses around `!this.options.disableaura`. Doesn't hurt to have them, but doesn't actually do anything either.

Reply:
aye thought that would be the case but just snipped it from somewhere else. So used to having my hands held with a linter